<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceConfig</class>
 <widget class="QDialog" name="DeviceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="radioTab">
      <attribute name="title">
       <string>Radio</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="typeLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>SDR type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="typeCombo">
         <property name="toolTip">
          <string>SDR radio type</string>
         </property>
         <item>
          <property name="text">
           <string>Generic (Xtal)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Generic (Si570)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Onnia Basic</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="xtalLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Xtal frerquency</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="xtalSpinBox">
         <property name="toolTip">
          <string>Fixed crystal frequency</string>
         </property>
         <property name="suffix">
          <string> MHz</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>500.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>7.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="fcorLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Frequency corr.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="fcorSpinBox">
         <property name="toolTip">
          <string>Frequency correction</string>
         </property>
         <property name="suffix">
          <string> ppm</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-500.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="sdrDevLabel">
         <property name="text">
          <string>Audio device</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="sdrDevCombo">
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Sound card device used for SDR</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="sdrRateLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Sample rate</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="sdrRateCombo">
         <property name="toolTip">
          <string>Sample rate used by the SDR device</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="sdrFormatLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Sample format</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="sdrFormatCombo">
         <property name="toolTip">
          <string>Sample format used by the SDR device</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="soundTab">
      <attribute name="title">
       <string>Sound card</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="aoutDevLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Speaker out</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="aoutDevCombo">
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Sound card device used for audio output</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="aoutRateLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Sample rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="aoutRateCombo">
         <property name="toolTip">
          <string>Sample rate for audio output</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="aoutFormatLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Sample format</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="aoutFormatCombo">
         <property name="toolTip">
          <string>Sample format fort audio output</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="ainDevLabel">
         <property name="text">
          <string>Microphone in</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="ainDevCombo">
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Sound card device used for audio input</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="ainRateLabel">
         <property name="text">
          <string>Sample rate</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="ainFormatLabel">
         <property name="text">
          <string>Sample format</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="ainRateCombo">
         <property name="toolTip">
          <string>Sample rate for audio input</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="ainFormatCombo">
         <property name="toolTip">
          <string>Sample format fort audio input</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeviceConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
